tests/modules/nf-core/windowmasker/convert/test.yml

```diff
@@ -0,0 +1,11 @@
+- name: windowmasker convert test_windowmasker_convert
+  command: nextflow run ./tests/modules/nf-core/windowmasker/convert -entry test_windowmasker_convert -c ./tests/config/nextflow.config -c ./tests/modules/nf-core/windowmasker/convert/nextflow.config
+  tags:
+    - windowmasker/convert
+    - windowmasker
+  files:
```

Comment:
As your module has the potential to output 3 (4?) different formats, you'll need a test per output.

See, this as an example:
https://github.com/nf-core/modules/blob/master/tests/modules/nf-core/bamtools/convert/test.yml